test/tests/performance/pbs_sched_perf.py

```diff
@@ -249,3 +249,41 @@ def test_pset_fuzzy_perf(self):
                         'Optimization was not faster')
         self.perf_test_result(((cycle1_time / cycle2_time) * 100),
                               "optimized_percentage", "percentage")
+
+    @timeout(10000)
+    def test_many_jobs_with_calendaring(self):
+        """
+        Performance test for when there are many jobs and calendaring is on
+        """
+        # Turn strict ordering on and backfill_depth=20
+        a = {'strict_ordering': 'True'}
+        self.server.manager(MGR_CMD_SET, MGR_OBJ_SERVER,
+                            {'backfill_depth': '20'})
+
+        self.server.manager(MGR_CMD_SET, MGR_OBJ_SERVER,
+                            {'scheduling': 'False'})
+        jids = []
+
+        # Submit around 10k jobs
+        chunk_size = 100
+        total_jobs = 10000
+        while total_jobs > 0:
+            for i in range(chunk_size):
```

Comment:
Why not do a range(10000)?

Reply:
I wanted to submit a mix of large and small jobs and I also wanted to submit a small number of large jobs and a large number of smaller jobs. This is just how I was able to automate the number of chunks to be in the range of 1 to 100 (chunk_size) and still submit the jobs in the pattern that I wanted.

Reply:
Oh, I didn't notice the jobs changing.  Yeah, different sizes of jobs is a good idea.

Reply:
oh, good info, so I should perhaps also do a mix of users